recipes/_install.rb

```diff
@@ -83,10 +83,15 @@ def nsp
 
 windows_password = CernerSplunk::DataBag.load(node['splunk']['windows_password'], secret: node['splunk']['data_bag_secret'])
 
+run_command = "#{node['splunk']['cmd']} help commands --accept-license --answer-yes --no-prompt"
+if Gem::Version.new(nsp['version']) >= Gem::Version.new('7.2.0')
+  run_command += " --seed-passwd 'changeme'"
```

Comment:
Ideally, I still think this should be the configured admin password (if there is one) rather than hard coding to 'changeme' and then letting the _generate_password recipe update it later on in the chef run. Functionally it all ends up the same though.  @acharlieh @nmattam thoughts?

Reply:
So I kinda agree with @gravesb on this, as some of the point of this feature I feel was to eliminate the changeme password even from the the very first start of the software and doing this feels like it kinda bypasses the control that was intended.

That said, arguably this cookbook for years has helped ensure that there was no default password being used, and (without reviewing the rest of this proposed change), it seems that it will continue to do so. So it might be an acceptable transitionary point for the time being, until say there could be a feature to work towards eliminating the `/etc/splunk/password` file :) 

I will say though, if you do keep the route of hardcoding the seed password to changeme, I would argue this is not a sensitive resource. (changeme is a well known default, and this is an open source project...) and should something go sideways with this command, non-sensitive resources are easier to debug.

Reply:
I agree with your suggestions but if we plan to use the admin_password directly for splunk-first-run for Splunk 7.2, We have to add a condition not to execute the _generate_password.rb for Splunk 7.2 and use almost all code from the _genereate_password.rb  except change password for Splunk 7.2.x  like
    1) creating  /etc/splunk/password
    2) adding new_password to file.
Instead of duplication, until all our environment upgraded to 7.2, we can use this way. Once our environment is upgraded we come up with an efficient approach.

Reply:
I'm ok with removing the sensitive true flag then and we'll plan to circle back later and rework the _generate_password recipe. Might be good to throw a TODO comment in here as a reminder.

Reply:
Removed sensitive true flag from splunk-first-run (_install.rb) and added TODO comment.